stdlib/public/core/Policy.swift

```diff
@@ -15,13 +15,33 @@
 //===----------------------------------------------------------------------===//
 // Standardized uninhabited type
 //===----------------------------------------------------------------------===//
-/// The return type of functions that do not return normally, that is, a type
-/// with no values.
+/// The return type of functions that cannot return a value.
+///
+/// As an uninhabited type, `Never` has no values and cannot be constructed 
```

Comment:
We should be careful to define the term `uninhabited` as that's type-theoretic jargon.

Reply:
Rephrased to remove this word.